share/locale/musescore_ru.ts

```diff
@@ -19549,12 +19549,12 @@ Fret %1 on strings %2</source>
     <message>
         <location filename="../../src/importexport/guitarpro/internal/gtp/gpconverter.cpp" line="1079"/>
         <source>Music by %1</source>
-        <translation>Автор музыки:</translation>
+        <translation>Автор музыки %1</translation>
```

Comment:
This one looks like the translator intended "Author of the music: %1", so the colon is still necessary. Same for "words by".

Reply:
Added the colon back

Reply:
> The original string doesn't have a colon, so the translation shouldn't either

I don't fully agree. If Russian doesn't have a way to translate `Music by …` (or maybe there is a literal translation but it's not the common way of saying it), and they want to translate it with `Author of the music: …`, that's okay.

Reply:
As said: I added it back